packages/wrapper/src/vscode/localeLoader.ts

```diff
@@ -59,3 +60,5 @@ export const initLocalLoader = async () => {
         }
     }
 };
+/** @deprecated use `initLocaleLoader` instead */
+export const initLocalLoader = initLocaleLoader;
```

Comment:
Thank you for spotting the typo in the function name. There is no need to deprecate the old name, just remove it.

Reply:
I've got that updated 👍